native/mod_manager/mod_manager.c

```diff
@@ -993,7 +993,7 @@ static char *process_config(request_rec *r, char **ptr, int *errtype)
     struct cluster_host *phost;
 
     int i = 0;
-    int id;
+    unsigned int id = 0;
```

Comment:
The `int` part can be dropped.

Reply:
most of the Apache httpd source codes are using unsigned int, I am not sure what is best.

Reply:
It is the same https://en.cppreference.com/w/c/language/arithmetic_types. I've seen they are using it with the `int` part but it's unnecessary and with a space.

Reply:
Actually there are more unsigned int in the code, I would prefer to fix all of them or make sure we use unsigned int everywhere. Mixing style is probably a bad idea...
So we have 2 ways:
1 - leave this PR as it is and fix later a separated PR where we fix all the unsigned int.
2 - fix this PR and do a later one fixing all the unsigned int remaining occurrences.

Reply:
You are right. I think out of those two options the first one is maybe better. There is also a third option, use `unsigned int` in all cases and let it be like that, which is also ok (I would prefer `unsigned`, but that's just me).

Reply:
> 1 - leave this PR as it is and fix later a separated PR where we fix all the unsigned int.

Yes - also lets follow best SCM practices that discourage from mixing functionality in a single commit with style changes.

> 2 - fix this PR 

Just fix to get the CI passing ;-) which is still failing.

> and do a later one fixing all the unsigned int remaining occurrences.

Yeah - ideally with a code style in CI fix - is that possible @jajik with what tooling we have?

Reply:
I am afraid that that is not possible, but I'll have a look. In both cases, I can take care of it.

Reply:
> I am afraid that that is not possible, but I'll have a look. In both cases, I can take care of it.

I couldn't find it either... worst case scenario just run `[[ $(git grep "unsigned int" | wc -l) -eq "0" ]] || exit 1` :-)